spec/amp-tag-addendum.md

```diff
@@ -0,0 +1,174 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AMP HTML Tag Addendum
+## Overview
+AMP HTML allows for most HTML5 standard tags and a few additional tags specific to the AMP runtime. The [AMP Spec](amp-html-format.md) broadly defines the set of tags that are disallowed. AMP Validator implementations, however, must be implemented using a tag whitelist. This addendum lists the set of tags which an AMP Validator should whitelist.
+
+## HTML5 Tag Whitelist
+Below we list the allowed tags in the order in which they are appear in the HTML5 spec in section 4 [The Elements of HTML](http://www.w3.org/TR/html5/single-page.html#html-elements).
+
+### 4.1 The root element
+4.1.1 `<html>`
+### 4.2 Document metadata
+4.2.1 `<head>`  
+4.2.2 `<title>`  
+4.2.4 `<link>`  
+4.2.5 `<meta>`  
+4.2.6 `<style>`  
```

Comment:
"amp-custom" is an attribute that a `<style>` required to have. I.e. 

```
<style amp-custom>
```

Validator already supports this rule, we just need to add a comment that says something like "Must be only one and have `amp-custom` attribute"